### DISCUSSION TITLE
firefox: cannot access Android phone device/data

### DISCUSSION THREAD

**Discussion by gitsteff** [netblue30/firejail#5262] [Q&A]
Hi.
I want to make the pictures from my android phone available to the firejailed firefox.
The phone is mounted via mtp:
`mtp://HMD_Global_Nokia_8.1_PNXG...../`
This is the mount belonging to it:
`gvfsd-fuse on /run/user/1000/gvfs type fuse.gvfsd-fuse (rw,nosuid,nodev,relatime,user_id=1000,group_id=1000)`

Does anyone know how to do it?
Thanks.

**Comment by rusty-snake** [comment]
Assuming it shows up with `ls /run/user/1000/gvfs/foobar`. You should try `whitelist ${RUNUSER}/gvfs` in firefox.local and mount it using `allow-root`/`allow-other`.

**Reply by gitsteff** [reply]
Your assumption about where it shows up was right, I did a new /etc/firejail/firefox.local with
`whitelist ${RUNUSER}/gvfs`
inside. Still I get the following error message. I couldnt manage it to change those mount parameters:
`gvfsd-fuse on /run/user/1000/gvfs type fuse.gvfsd-fuse (rw,nosuid,nodev,relatime,user_id=1000,group_id=1000)`
I guess, the error is because of the use of my user_id and group_id only and not including root, right?

Do you have an idea where to configure this? It all happens automaticlly. This VM is running manjaro under kvm.


```
firejail firefox
Reading profile /etc/firejail/firefox.profile
Reading profile /etc/firejail/firefox.local
Reading profile /etc/firejail/whitelist-usr-share-common.inc
Reading profile /etc/firejail/firefox-common.profile
Reading profile /etc/firejail/disable-common.inc
Reading profile /etc/firejail/disable-devel.inc
Reading profile /etc/firejail/disable-exec.inc
Reading profile /etc/firejail/disable-interpreters.inc
Reading profile /etc/firejail/disable-proc.inc
Reading profile /etc/firejail/disable-programs.inc
Reading profile /etc/firejail/whitelist-common.inc
Reading profile /etc/firejail/whitelist-run-common.inc
Reading profile /etc/firejail/whitelist-runuser-common.inc
Reading profile /etc/firejail/whitelist-var-common.inc
Seccomp list in: !chroot, check list: @default-keep, prelist: unknown,
Parent pid 10106, child pid 10109
Warning: An abstract unix socket for session D-BUS might still be available. Use --net or remove unix from --protocol set.
Error fstat: fs_whitelist.c:156 whitelist_file: Permission denied
Error: proc 10106 cannot sync with peer: unexpected EOF
Peer 10109 unexpectedly exited with status 1
```

**Reply by rusty-snake** [reply]
`/etc/fuse.conf` IIRC
